openpype/hosts/max/plugins/publish/extract_thumbnail.py

```diff
@@ -23,22 +21,26 @@ def process(self, instance):
         self.log.debug(
```

Comment:
I understand the `tmp_staging` here but I think the publishing doesn't clean this up currently after a succesful publish. Or does it? 

It'd be much more trivial to instead write into the same temp directory but just change the output file name of this particular extraction, yes?

```python
    staging_dir = self.staging_dir(instance)
    filepath = os.path.join(staging_dir, f"{instance.name}_thumbnail")
```